core/api/src/graphql/public/root/query/ln-invoice-payment-status.ts

```diff
@@ -13,20 +13,28 @@ const LnInvoicePaymentStatusQuery = GT.Field({
   },
   resolve: async (_, args) => {
     const { paymentRequest } = args.input
-    if (paymentRequest instanceof Error) throw paymentRequest
+    if (paymentRequest instanceof Error) {
```

Comment:
Same here, I think we usually throw our errors for queries and return for mutations

Reply:
type payload contains error.. really not sure if was on purpose or because it is used to in subscription... do you remember?